db/migrate/20151018090641_create_persistent_volume_claim.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePersistentVolumeClaim < ActiveRecord::Migration
+  def change
+    create_table :persistent_volume_claims do |t|
+      t.belongs_to :ems, :type => :bigint
+      t.belongs_to :persistent_volume, :type => :bigint
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :ems_created_on
+      t.timestamp  :created_on
+      t.string     :resource_version
+      t.string     :desired_access_modes, :array => true, :default => []
```

Comment:
Better switch it to t.text because of size limitations.
See -> http://stackoverflow.com/questions/3354330/difference-between-string-and-text-in-rails